### PR TITLE
Move cgminer and ccminer into Unofficial Miners.

### LINKED ISSUE #310: Update references to cgminer and ccminer
https://docs.decred.org/mining/proof-of-work/#select-and-download-mining-software

These miners are no longer part of the decred project, gominer has replaced them.

cg/cc miner can still be used to mine DCR, however decred developers will no longer be distributing them and the repos have been emptied. Therefore we need to update the links to these miners and move them to the "unofficial" section

### PR DESCRIPTION
**Pull request by jholdstock**
Closes #310

### CODE REVIEW

File: docs/mining/proof-of-work.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ are made. These coins are then split up as follows:
 
 Subsidy | Party
 ---     | ---
-`60%`   | PoW Miners
-`30%`   | PoS Voters
-`10%`   | Decred development subsidy
+60%   | PoW Miners
```
Comment (jcvernaleo) [inline docs/mining/proof-of-work.md]: This isn't really related to the mining changes, is it?

File: docs/mining/proof-of-work.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Pool mining distributes shares based on blocks found so you can earn a steady am
 
 ---
 
-## Obtain a Decred Address to Receive Funds With
+## ** Obtain a Decred Address to Receive Funds With **
```
Comment (jcvernaleo) [inline docs/mining/proof-of-work.md]: This change also seems unrelated.

File: docs/mining/proof-of-work.md [hunk 3]
```diff
@@ -69,30 +69,21 @@ GPU drivers usually contain the libraries needed for mining.  If you have diffic
 
 ## **<i class="fa fa-download"></i> Select and Download Mining Software **
 
-> Official Decred Mining Software Builds
+### Official Decred Miner (gominer)
 
-#### **[<i class="fa fa-github"></i> gominer](https://github.com/decred/gominer)**
-
-It is recommended that everyone use gominer for Decred. This is the easiest miner to setup and start mining. Currently, the latest version of gominer is **[<i class="fa fa-github"></i>v1.0.0](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **[https://github.com/decred/decred-binaries/releases/tag/v1.0.0](https://github.com/decred/decred-binaries/releases/tag/v1.0.0)**
+It is recommended that everyone use gominer for Decred. This is the easiest miner to setup and start mining. Currently, the latest version of gominer is **<i class="fa fa-github"></i> [v1.0.0](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **[https://github.com/decred/decred-binaries/releases/tag/v1.0.0](https://github.com/decred/decred-binaries/releases/tag/v1.0.0)**
```
Comment (jcvernaleo) [inline docs/mining/proof-of-work.md]: I realize it always said this, but do we really want to say it so strongly?  The other miners are good too, gominer is just the one we can actually help with and make changes to.  Maybe we should say 'encouraged' instead of 'recommended'?  I agree that it is easiest to get started with though.  So not really sure.  My point is just that this sounds too strongly in favor of gominer instead of the others (we don't actively discuourage the other miners, we just encourage gominer).  Any thoughts?

File: docs/mining/proof-of-work.md (hunk 1 above)
Comment (jholdstock) [inline docs/mining/proof-of-work.md]: Nope just a small improvement in appearance and in making our use of markdown more correct/consistent. I will remove this if you wish

File: docs/mining/proof-of-work.md (hunk 2 above)
Comment (jholdstock) [inline docs/mining/proof-of-work.md]: Nope just a small improvement in appearance and in making our use of markdown more correct/consistent. I will remove this if you wish

File: docs/mining/proof-of-work.md (hunk 3 above)
Comment (jholdstock) [inline docs/mining/proof-of-work.md]: Agreed the wording could be a bit softer. How about:

"Gominer is the official Decred miner developed and supported by the Decred team. It is the easiest miner to setup and start mining and so it is recommended for most users."

File: docs/mining/proof-of-work.md (hunk 1 above)
Comment (jcvernaleo) [inline docs/mining/proof-of-work.md]: Leaving them in is fine but then please mention them in the commit.

File: docs/mining/proof-of-work.md (hunk 2 above)
Comment (jcvernaleo) [inline docs/mining/proof-of-work.md]: same as above

File: docs/mining/proof-of-work.md (hunk 3 above)
Comment (jcvernaleo) [inline docs/mining/proof-of-work.md]: I like it.

File: docs/mining/proof-of-work.md (hunk 1 above)
Comment (jholdstock) [inline docs/mining/proof-of-work.md]: Cool, will do in future.

**Comment by jholdstock** [comment]
Softened the recommendation wording, I think this one is good now
